src/main/java/com/msgilligan/bitcoin/BTC.java

```diff
@@ -8,7 +8,7 @@
  * Utility class for converting from satoshis (BigInteger) to bitcoins (BigDecimal) and vice-versa
  */
 public class BTC  {
-    public static final long satoshisPerBitcoin = 100000000;
+    public static final long satoshisPerBitcoin = 100000000L;
```

Comment:
Integer literals default to type `int`, but an `int` literal will be *implicitly* converted to a `long`if assigned to a `long` field. Apparently, the `L` is required if the integer literal is too large, so that's why I used it in at least one place.